bazel/repository_locations.bzl

```diff
@@ -1099,4 +1099,17 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2018-12-18",
         cpe = "N/A",
     ),
+    com_github_fdio_vpp_vcl = dict(
+        project_name = "VPP Comms Library",
+        project_desc = "FD.io Vector Packet Processor (VPP) Comms Library",
+        project_url = "https://fd.io/",
+        version = "895def45c82ea5d901987bc3049c8d1cc1c1da66",
```

Comment:
There have been over 1000 commits since the last release - https://github.com/FDio/vpp/releases. Any idea when VPP will cut another release?

Reply:
Scores on OSSF Scorecard not great

[ossf_scorecard_vpp.txt](https://github.com/envoyproxy/envoy/files/7181749/ossf_scorecard_vpp.txt)

@envoyproxy/dependency-shepherds wondering if we need to update the [policy](https://github.com/envoyproxy/envoy/blob/main/DEPENDENCY_POLICY.md) to lower requirements for contrib extensions?

Reply:
> There have been over 1000 commits since the last release - https://github.com/FDio/vpp/releases. Any idea when VPP will cut another release?

Within a month, we're planning a 21.10 release.

Reply:
> Scores on OSSF Scorecard not great
> 
> [ossf_scorecard_vpp.txt](https://github.com/envoyproxy/envoy/files/7181749/ossf_scorecard_vpp.txt)
> 
> @envoyproxy/dependency-shepherds wondering if we need to update the [policy](https://github.com/envoyproxy/envoy/blob/main/DEPENDENCY_POLICY.md) to lower requirements for contrib extensions?

Hm, I suspect this is data generated based on our github mirror. We're using gerrit for development gerrit.fd.io and the main repo is git.fd.io

Reply:
Yes, I ran the tool against Github. I received errors trying it again main repo and found out the OSSF Scorecard only supports Github - https://github.com/ossf/scorecard/blob/main/repos/repo_url.go#L96-L109. Sigh

Reply:
Oh well, sorry about that. Is there any information we could try to glean from https://gerrit.fd.io/r/q/project:vpp or https://git.fd.io/vpp? And for context, fd.io is a Linux Foundation umbrella project that, among others, includes VPP. 

Should note that I'm butchering VPP's build system to make sure that it only builds VCL (a library). Otherwise, if we were to aim for a full build, we'd be incrementing Envoy's build time by too much as VPP is a pretty large project. Also, by default libvppcom is dynamically linked, let me know if that's an issue.

Reply:
I think for contrib we probably care less about the score card number.
With respect to dynamic linking, I think this is fine too. We just need to document how to install vpp binaries.